podcasts/Onboarding/Login/LoginLandingView.swift

```diff
@@ -0,0 +1,255 @@
+import SwiftUI
+
+struct LoginLandingView: View {
+    @EnvironmentObject var theme: Theme
+    @ObservedObject var coordinator: LoginCoordinator
+
+    var body: some View {
+        ProportionalValueFrameCalculator {
+            LoginLandingContent(coordinator: coordinator)
+        }
+    }
+
+    func configure(controller: UIViewController) {
+        let imageView = ThemeableImageView(frame: .zero)
+        imageView.imageNameFunc = AppTheme.pcLogoSmallHorizontalForBackgroundImageName
+        imageView.accessibilityLabel = L10n.accountLogin
+        controller.navigationItem.titleView = imageView
+
+        controller.navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Not Now", style: .plain, target: coordinator, action: #selector(LoginCoordinator.dismissTapped))
+    }
+}
+
+private struct LoginLandingContent: View {
+    @EnvironmentObject var theme: Theme
+    @ObservedObject var coordinator: LoginCoordinator
+    @ProportionalValue(with: .height) var calculatedHeight: Double
+
+    init(coordinator: LoginCoordinator) {
+        self.coordinator = coordinator
+
+        // Find the value that will appear the lowest, and use that to calculate the
+        // "total view height" since the actual view height doesn't account correctly
+        let maxModel = models.max {
+            $0.y < $1.y
+        }
+
+        calculatedHeight = maxModel?.y ?? 0
+    }
+
+    var body: some View {
+        ZStack(alignment: .top) {
+            AppTheme.color(for: .primaryUi01, theme: theme).ignoresSafeArea()
+            LoginHeader(models: models, topPadding: Config.padding)
+
+            ScrollViewIfNeeded {
+                VStack {
+                    // Title and Subtitle
+                    VStack(spacing: 8) {
+                        LoginLabel("Discover your next favorite podcast", for: .title)
+                        LoginLabel("Create an account to sync your listening experience across all your devices.", for: .subtitle)
+                    }
+
+                    Spacer()
+
+                    LoginButtons(coordinator: coordinator)
+                }
+                .padding([.leading, .trailing], Config.padding)
+                .padding(.top, calculatedHeight + (Config.padding * 2))
+                .padding(.bottom)
+            }
+        }
+    }
+
+    // Cover Models 📸
+    private let models: [CoverModel] = [
+        CoverModel(imageName: "login-cover-1", size: 0.26, x: -0.1973, y: 0.2606),
```

Comment:
Do you think it would be bad if we loaded those covert arts?

The reason I ask is that these 7 covers will probably add some MBs into the final app version for something that is not critical.

We also need to take into account that if we load them from remote they might change (not sure if that would be an issue).